api_app/analyzers_manager/migrations/0160_analyzer_config_joesandboxfile.py

```diff
@@ -0,0 +1,182 @@
+from django.db import migrations
+from django.db.models.fields.related_descriptors import (
+    ForwardManyToOneDescriptor,
+    ForwardOneToOneDescriptor,
+    ManyToManyDescriptor,
+    ReverseManyToOneDescriptor,
+    ReverseOneToOneDescriptor,
+)
+
+plugin = {
+    "python_module": {
+        "health_check_schedule": None,
+        "update_schedule": None,
+        "module": "joesandbox_file.JoeSandboxFile",
+        "base_path": "api_app.analyzers_manager.file_analyzers",
+    },
+    "name": "JoeSandboxFile",
+    "description": "[JoeSandboxFile](https://www.joesandbox.com/) is a comprehensive malware analysis tool, which can be used to perform deep malware analysis, on various platforms such as windows, macos, linux, android.",
+    "disabled": False,
+    "soft_time_limit": 1000,
+    "routing_key": "default",
+    "health_check_status": True,
+    "type": "file",
+    "docker_based": False,
+    "maximum_tlp": "RED",
+    "observable_supported": [],
+    "supported_filetypes": [
+        "application/vnd.android.package-archive",
+        "application/zip",
+        "multipart/x-zip",
+        "application/java-archive",
+        "application/vnd.microsoft.portable-executable",
+        "application/vnd.tcpdump.pcap",
+        "application/pdf",
+        "application/vnd.ms-excel",
+        "application/excel",
+        "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet",
+        "text/csv",
+        "application/vnd.openxmlformats-officedocument.presentationml.presentation",
+        "application/msword",
+        "application/vnd.openxmlformats-officedocument.wordprocessingml.document",
+        "application/vnd.ms-powerpoint",
+        "application/vnd.ms-office",
+        "application/x-binary",
+        "application/x-macbinary",
+        "application/mac-binary",
+        "application/x-mach-binary",
+        "application/x-executable",
+        "text/x-java",
+    ],
+    "run_hash": False,
+    "run_hash_type": "",
+    "not_supported_filetypes": [],
+    "mapping_data_model": {},
+    "model": "analyzers_manager.AnalyzerConfig",
+}
+
+params = [
+    {
+        "python_module": {
+            "module": "joesandbox_file.JoeSandboxFile",
+            "base_path": "api_app.analyzers_manager.file_analyzers",
+        },
+        "name": "url",
+        "type": "str",
+        "description": "URL for your JoeSandbox Instance.",
+        "is_secret": False,
+        "required": True,
+    },
+    {
+        "python_module": {
+            "module": "joesandbox_file.JoeSandboxFile",
+            "base_path": "api_app.analyzers_manager.file_analyzers",
+        },
+        "name": "system_to_use",
+        "type": "str",
+        "description": 'System to use for analysis. Defaults to "lnxubuntu20". Read the Intelowl Usage docs, to get more info. on what systems are available.',
```

Comment:
```suggestion
        "description": 'System to use for analysis. Defaults to "lnxubuntu20". Read the Intelowl Usage docs, to get more info on what systems are available.',
```
corrected typo. Plus, this is the name of the VM, right? I think there should not be a default because, let's say I am analyzing a Windows EXE and you force it to ubuntu. It won't work.

Reply:
1. yup this is the name of the system on which the analysis would be run. I'll add each available name with brief description, for the users in the usage docs. 
2. Though, there is an option to not set it at all, basically `auto` mode kicks in, but during my testing it didn't work, so I'd set a default machine to run a basic analysis like URL analysis or some basic pcap analysis. 

    But yeah it makes sense to not keep it as default.